src/transformers/models/seaformer/configuration_seaformer.py

```diff
@@ -0,0 +1,192 @@
+            Dimension of each of the encoder blocks.
+        patch_sizes (`List[int]`, *optional*, defaults to [7, 3, 3, 3]):
+            Patch size before each encoder block.
+        strides (`List[int]`, *optional*, defaults to [4, 2, 2, 2]):
+            Stride before each encoder block.
+        num_attention_heads (`List[int]`, *optional*, defaults to [1, 2, 5, 8]):
+            Number of attention heads for each attention layer in each block of the Transformer encoder.
+        mlp_ratios (`List[int]`, *optional*, defaults to [4, 4, 4, 4]):
+            Ratio of the size of the hidden layer compared to the size of the input layer of the Mix FFNs in the
+            encoder blocks.
+        hidden_act (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
+            `"relu"`, `"selu"` and `"gelu_new"` are supported.
+        hidden_dropout_prob (`float`, *optional*, defaults to 0.0):
+            The dropout probability for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (`float`, *optional*, defaults to 0.0):
+            The dropout ratio for the attention probabilities.
+        classifier_dropout_prob (`float`, *optional*, defaults to 0.1):
+            The dropout probability before the classification head.
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        drop_path_rate (`float`, *optional*, defaults to 0.1):
+            The dropout probability for stochastic depth, used in the blocks of the Transformer encoder.
+        layer_norm_eps (`float`, *optional*, defaults to 1e-6):
+            The epsilon used by the layer normalization layers.
+        decoder_hidden_size (`int`, *optional*, defaults to 256):
+            The dimension of the all-MLP decode head.
+        semantic_loss_ignore_index (`int`, *optional*, defaults to 255):
+            The index that is ignored by the loss function of the semantic segmentation model.
+
+    Example:
+
+    ```python
+    >>> from transformers import SeaformerModel, SeaformerConfig
+
+    >>> # Initializing a Seaformer nvidia/seaformer-b0-finetuned-ade-512-512 style configuration
+    >>> configuration = SeaformerConfig()
+
+    >>> # Initializing a model from the nvidia/seaformer-b0-finetuned-ade-512-512 style configuration
+    >>> model = SeaformerModel(configuration)
+
+    >>> # Accessing the model configuration
+    >>> configuration = model.config
+    ```"""
+    model_type = "seaformer"
+
+    def __init__(
+        self,
+        num_channels=3,
+        num_encoder_blocks=3,
+        depths=[3, 3, 3],
+        num_labels = 150,
+        channels = [32, 64, 128, 192, 256, 320],
+        cfgs = [
```

Comment:
Some of the attributes are not in the docstrings above, could you update it and include all attributes (cfgs, is_dw)?

We also make the attribute names as descriptive as possible for ease of use (e.g. is_dw -> is_depthwise)